test/unit_tests/test_Sources_opt.jl

```diff
@@ -90,7 +90,7 @@ end
 ## Sedimentation.jl
```

Comment:
Let's add an additional opt test for Coalescence where the user specifies a normalization factor instead of the default

Reply:
It's done now!